src/ints/xms.cpp

```diff
@@ -102,392 +110,752 @@ struct XMS_MemMove{
 #pragma pack ()
 #endif
 
+// ***************************************************************************
+// Variables
+// ***************************************************************************
+
+static struct {
+	bool enable_global = false;
+
+	uint32_t enable_local_count = 0;
```

Comment:
The "enable" prefix is a boolean action, so as a reader see this as says  *"Should we enable the local count or disable the local count?"*

I see the task is to count how many times the A20 line has been enabled; so "enable" needs to become an adjective instead of an action. 

Maybe `num_times_enabled` or `enablement_tally`?

Reply:
Changed to `num_times_enabled`.